src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Concatenate.cs

```diff
@@ -25,7 +25,7 @@ internal sealed class ConcatenateFunction : BuiltinFunction
         public override bool SupportsParamCoercion => true;
 
         public ConcatenateFunction()
-            : base("Concatenate", TexlStrings.AboutConcatenate, FunctionCategories.Text, DType.String, 0, 0, int.MaxValue)
+            : base("Concatenate", TexlStrings.AboutConcatenate, FunctionCategories.Text, DType.String, 0, 1, int.MaxValue)
```

Comment:
>1 [](http://example.com/codeflow?start=106&length=1)

This is a breaking change. Today `Concatenate()` is a valid expression (not sure why anyone would do it, but it's valid today) that returns an empty string.